src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusHelpers.hpp

```diff
@@ -82,85 +81,102 @@ T set_bc_when_char_speed_is_negative(const T& rhs_char_dt_u,
 // Eq. (2.20) of Kidder, Scheel & Teukolsky (KST) [gr-qc/0105031v1]
 // Compute R_{ij} from D_(k,i,j) = (1/2) \partial_k g_(ij)
 template <size_t VolumeDim, typename Frame, typename DataType>
-tnsr::ii<DataType, VolumeDim, Frame> spatial_ricci_tensor_from_KST_vars(
-    const tnsr::ijj<DataType, VolumeDim, Frame>& kst_var_D,
-    const tnsr::ijkk<DataType, VolumeDim, Frame>& d_kst_var_D,
+tnsr::ii<DataType, VolumeDim, Frame> ricci_tensor(
```

Comment:
Not sure, but I like the idea of moving this to Pointwisefunctions. Since the inputs to this method are specific to GH, I've moved it to `PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/Ricci.?pp?`. Will add a unit test shortly.